src/vector/platform/ElectronPlatform.tsx

```diff
@@ -305,7 +303,18 @@ export default class ElectronPlatform extends VectorBasePlatform {
         return this.ipc.call('setLanguage', preferredLangs);
     }
 
-    public setSpellCheckLanguages(preferredLangs: string[]) {
+    public setSpellCheckEnabled(enabled: boolean): void {
+        this.ipc.call('setSpellCheckEnabled', enabled).catch(error => {
+            logger.log("Failed to send setSpellCheckEnabled IPC to Electron");
+            logger.error(error);
+        });
+    }
+
+    public async getSpellCheckEnabled(): Promise<boolean> {
+        return this.ipc.call('getSpellCheckEnabled');
+    }
+
+    setSpellCheckLanguages(preferredLangs: string[]): void {
```

Comment:
this lost a `public`

```suggestion
    public setSpellCheckLanguages(preferredLangs: string[]): void {
```